documentation/modules/upgrading/proc-reenabling-FIPS-mode.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-reenabling-FIPS-mode-{context}']
+= Re-enabling FIPS mode
+
+[role="_abstract"]
+Until Strimzi 0.33, running on FIPS-enabled Apache Kafka clusters was possible only with the FIPS mode disabled using the `FIPS_MODE` environment variable.
+But from 0.33 on, Strimzi supports running on FIPS-enabled Kubernetes clusters without disabling the FIPS mode.
+If you run Strimzi on a FIPS-enabled Kubernetes cluster with the `FIPS_MODE` set to `disabled`, you can re-enable it by following this procedure.
+
+.Prerequisites
+
+* FIPS-enabled Kubernetes cluster
+* An existing Cluster Operator deployment with the `FIPS_MODE` environment variable set to `disabled`
+
+.Procedure
+
+. Upgrade Strimzi Cluster Operator to version 0.33 or newer but keep the `FIPS_MODE` environment variable set to `disabled`
+
+. If you initially deployed your cluster with a Strimzi version older than 0.30, it might still use the PKCS12 stores using old encryption and digest algorithms which are not supported with FIPS enabled.
+  To get the certificates recreated with the new algorithms, renew the Cluster and Clients CA.
+  
+.. For renewing the CAs generated by the Cluster Operator, use the link:{BookURLUsing}#proc-renewing-ca-certs-manually-str[annotation to trigger the renewal^].
+
+.. For renewing your own CAs, follow the docs about link:{BookURLUsing}#renewing-your-own-ca-certificates-str[renewing your own CA certificates^].
+
+. If you use SCRAM-SHA authentication, you have to check the password length of your users.
+  If they are less than 32 characters long, you will have to generate a new password for them.
+  
+.. You can do that by deleting the user secret and having the User Operator generate a new one with a new password with sufficient length.
+
+.. Alternatively, if you provided your password using the `.spec.authentication.password` section of the `KafkaUser` custom resource, you have to update the password in the Kubernetes Secret referenced here.
+   Don't forget to update your clients to use the new passwords.
+
+. Once all the certificates are using the correct algorithms and the SCRAM-SHA passwords are sufficiently long, you can now enable the FIPS mode.
+  You can do so by removing the environment variable `FIPS_MODE` from the Strimzi Cluster Operator deployment.
+  The operator will restart and will roll all the operands to re-enable the FIPS mode.
+  And once it is done, all your Kafka clusters will not run with the OpenJDK FIPS mode enabled.
```

Comment:
```suggestion
. Remove the `FIPS_MODE` environment variable from the Cluster Operator deployment.
+
You can use the `060-Deployment-strimzi-cluster-operator.yaml` installation file to make this change.

. Deploy the Cluster Operator:
+
[source,shell]
kubectl create -f install/cluster-operator -n myproject
+
This restarts the Cluster Operator and rolls all the operands to enable the FIPS mode.
After the restart is complete, all Kafka clusters now run with FIPS mode enabled.
```

Reply:
It said _not run_ here. I don't know if I'm reading the procedure incorrectly, but I'm assuming they can now run in FIPS mode where appropriate.

Reply:
I mostly followed the suggestion here. But left out the part about the exact files since that would not cover the other installation methods. So I think it is better to leave it to the users to resolve it in whatever way they used for installation.